src/main/java/uk/gov/pay/adminusers/resources/ServiceResource.java

```diff
@@ -13,14 +13,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import uk.gov.pay.adminusers.exception.ServiceNotFoundException;
-import uk.gov.pay.adminusers.exception.ValidationException;
 import uk.gov.pay.adminusers.model.*;
```

Comment:
Not related to this PR but noticed a few wild card imports `import uk.gov.pay.adminusers.model.*. Would be good if they can be removed in future PR

Reply:
Good spot, will remove in my next PR